installer/install.iss

```diff
@@ -1576,9 +1576,9 @@ end;
 
 // Procedure to create hardlinks for builtins. This procedure relies upon that
 // git-wrapper.exe is already copied to {app}\tmp.
-procedure CopyBuiltin(FileName:String);
+procedure HardlinkOrCopyGit(FileName:String,Builtin:Boolean);
 var
-    AppDir:String;
+    AppDir,GitTarget:String;
     LinkCreated:Boolean;
 begin
     if (not DeleteFile(FileName)) then begin
```

Comment:
Wouldn't it be more robust to execute `DeleteFile` only if the file exists? This would also fix the `git-lfs` case, where `DeleteFile` fails because there's no existing file under `/cmd/` and therefore no hardlink would be created.

Reply:
Oh! Absolutely!